app/services/WcoMetadataMapper.scala

```diff
@@ -29,5 +48,102 @@ class WcoMetadataMapper {
 
   def getDeclarationLrn(metaData: Any): Option[String] = self.declarationLrn(metaData)
 
-  def serialise(metaData: Any): String = self.toXml(metaData)
+  def serialise(metaData: Any): String = toXml(metaData)
+
+
+}
+
+object WcoMetadataMapper {
```

Comment:
As we have discussed, this logic should not be here as this WcoMetadataMapper is a `Facade` for a mapping strategy. In this case, we are leaking the Scala implementation of `GoodsItems` into the Mapper.

If you could change great, otherwise once it is in we should move the implementation to a mapping strategy.

Reply:
I've moved this now